Makefile

```diff
@@ -28,5 +31,11 @@ prepare:
 update:
 	glide update
 
-run:
-	cd bin && ./edgex-launch.sh
\ No newline at end of file
+docker: $(DOCKERS)
+
+docker_device_modbus_go:
+	docker build \
+		--label "git_sha=$(GIT_SHA)" \
```

Comment:
It doesn't look like GIT_SHA is being defined anywhere

Reply:
I was missing a build argument in .PHONY.
I pushed a minor change.

Reply:
The GIT_SHA should come as a global variable from the git commit hash.

Reply:
You'll still need to add a line like this one: https://github.com/edgexfoundry/device-random/blob/master/Makefile#L13

Reply:
thank you @mhall119 I have pushed that change now.